sov-modules/sov-modules-macros/src/dispatch/common.rs

```diff
@@ -110,8 +110,8 @@ impl<'a> StructDef<'a> {
             #[allow(non_camel_case_types)]
             // TODO we should not hardcode the serialization format inside the macro:
             // https://github.com/Sovereign-Labs/sovereign/issues/97
-            #[derive(borsh::BorshDeserialize, borsh::BorshSerialize)]
-            enum #enum_ident #impl_generics #where_clause {
+            #[derive(borsh::BorshDeserialize, borsh::BorshSerialize, ::core::fmt::Debug, PartialEq)]
+            pub enum #enum_ident #impl_generics #where_clause {
```

Comment:
Let's keep it private and use `Runtime::<C>::decode_call` for decoding

Reply:
Yep, makes sense!